test/fixtures/googleembeds.json

```diff
@@ -15,38 +15,40 @@
       "children": [
         {
           "type": "text",
-          "value": "Here’s an image with a QR Code: The QR Code resolves to the url of a Youtube clip. This image will be replaced by a Link+Image-style Embed. "
+          "value": "Here’s an image with a QR Code: The QR Code resolves to the url of a Youtube clip. This image will be replaced by a Link+Image-style Embed."
         }
       ]
     },
     {
-      "type": "paragraph",
+      "type": "embed",
```

Comment:
@trieloff can you take a 2nd look here? don't know if this is still correct.

Reply:
looks iffy: https://github.com/adobe/helix-pipeline/pull/878#discussion_r519642694

Reply:
No, it is correct.